aries-site/src/pages/whats-new.mdx

```diff
@@ -52,6 +52,84 @@ regarding either via [Slack](https://hpe.slack.com/archives/C04LMJWQT).
 
 <!-- BEGIN: What's New Entries -->
 
+## January 4 - January 21, 2022
+
+### "Big Things" progress updates
```

Comment:
```suggestion
### "Big Things" progress updates

```

Reply:
Also wondering if we should just call this "Roadmap Updates" and omit this sentence below